testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/db/SemconvStabilityUtil.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.testing.junit.db;
+
+import static io.opentelemetry.semconv.incubating.DbIncubatingAttributes.DB_CASSANDRA_TABLE;
+import static io.opentelemetry.semconv.incubating.DbIncubatingAttributes.DB_COSMOSDB_CONTAINER;
+import static io.opentelemetry.semconv.incubating.DbIncubatingAttributes.DB_MONGODB_COLLECTION;
+import static io.opentelemetry.semconv.incubating.DbIncubatingAttributes.DB_NAME;
+import static io.opentelemetry.semconv.incubating.DbIncubatingAttributes.DB_OPERATION;
+import static io.opentelemetry.semconv.incubating.DbIncubatingAttributes.DB_REDIS_DATABASE_INDEX;
+import static io.opentelemetry.semconv.incubating.DbIncubatingAttributes.DB_SQL_TABLE;
+import static io.opentelemetry.semconv.incubating.DbIncubatingAttributes.DB_STATEMENT;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.instrumentation.api.internal.SemconvStability;
+import java.util.HashMap;
+import java.util.Map;
+
+// until old database semconv are dropped in 3.0
+public class SemconvStabilityUtil {
+  private static final Map<AttributeKey<?>, AttributeKey<?>> oldToNewMap = new HashMap<>();
+  private static final AttributeKey<String> DB_NAMESPACE = AttributeKey.stringKey("db.namespace");
+  private static final AttributeKey<Long> DB_NAMESPACE_LONG = AttributeKey.longKey("db.namespace");
```

Comment:
`db.namespace` is always a String

Reply:
it was for DB_REDIS_DATABASE_INDEX.

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/11575/commits/98d20fc105078babce5f14fe2247590a83d5021a